tests/e2e/cucumber/features/integrations/search.feature

```diff
@@ -0,0 +1,53 @@
+Feature: link
+  As a user
+  I want to search for resources
+  So that I can find them quickly
+
+
+  Scenario: Search in personal spaces
+    Given "Admin" creates following users
+      | id    |
+      | Alice |
+    When "Alice" logs in
+    And "Alice" opens the "files" app
+    And "Alice" creates the following resources
+      | resource     | type   |
+      | folder       | folder |
+      | new-folder   | folder |
+      | FolDer       | folder |
+      | folderPublic | folder |
+      | public       | folder |
+    When "Alice" searches "fol" using the global search
+    Then following resources should be displayed in the search list for user "Alice"
+      | resource     |
+      | folder       |
+      | new-folder   |
+      | FolDer       |
+      | folderPublic |
+    But following resource should not be listed in the search list for user "Alice"
+      | resource |
+      | public   |
```

Comment:
@ScharfViktor this step fails because the folder is displayed in the search list, what to do for this now?